scalableminds/webknossos
# Make layer settings visually consistent

### URL of deployed dev instance (used for testing):
- https://makelayersettingsvisualconsistent.webknossos.xyz/

### Steps to test:
- Open a tracing with more than one color layer.
- The color layers should be able to be rearranged.
- The volume and skeleton layers should also have a drag handle with less opacity. The drag handle should be disabled and a tooltip should tell that this layer cannot be moved.

### TODOs:

### Issues:
- no issue exists for this

------
(Please delete unneeded items, merge only when none are left open)
